corehq/ex-submodules/casexml/apps/case/tests/test_multimedia.py

```diff
@@ -300,3 +302,34 @@ def test_sync_log_invalidation_bug(self):
         # this used to fail before we fixed http://manage.dimagi.com/default.asp?158373
         self._doSubmitUpdateWithMultimedia(new_attachments=['commcare_logo_file'], removes=[],
                                            sync_token=sync_log._id)
+
+
+@skip("Case attachments don't work with S3 API yet")
+class CaseMultimediaS3DBTest(BaseCaseMultimediaTest):
+    """
+    Tests new attachments for cases and case properties
+    Spec: https://github.com/dimagi/commcare/wiki/CaseAttachmentAPI
+    """
+
+    def setUp(self):
+        super(CaseMultimediaS3DBTest, self).setUp()
+        with trap_extra_setup(AttributeError, msg="S3_BLOB_DB_SETTINGS not configured"):
+            config = settings.S3_BLOB_DB_SETTINGS
+            self.s3db = TemporaryS3BlobDB(config)
+            assert get_blob_db() is self.s3db, (get_blob_db(), self.s3db)
+
+    def tearDown(self):
+        self.s3db.close()
+        super(CaseMultimediaS3DBTest, self).tearDown()
+
+    @run_with_all_backends
+    def test_case_attachment(self):
+        single_attach = 'fruity_file'
+        xform, case = self._doCreateCaseWithMultimedia(attachments=[single_attach])
```

Comment:
This sends attachments to [`submit_form_locally`](https://github.com/dimagi/commcare-hq/blob/9a9de02b8bcbffb913ca0671e11cca1396ad20d5/corehq/ex-submodules/casexml/apps/case/tests/test_multimedia.py#L94-L100), which I would expect to work. I thought case attachments were working, but still stored in couch. Is that not true?